carla/recourse_methods/autoencoder/models.py

```diff
@@ -0,0 +1,150 @@
+import os
+from typing import Callable, List, Optional
+
+import numpy as np
+from keras import backend as K
+from keras.layers import Dense, Input
+from keras.models import Model, Sequential
+
+
+def layers_valid(layers: List) -> bool:
```

Comment:
I thought that we maybe define later additional Autoencoder classes, so we could re-use this method there. But until then, it would make more sense to have this inside the class.